packages/clay-css/gulpfile.js

```diff
@@ -194,6 +195,19 @@ gulp.task(
 	}
 );
 
+gulp.task('copy:licenses', function() {
+	var newLicenseDir = path.join('.', 'LICENSES');
```

Comment:
Wouldn't this work too?

```js
var newLicenseDir = 'LICENSES';
```

Because for most intents and purposes, `LICENSES` and `./LICENSES` is the same thing.

Reply:
Absolutely, this makes total sense.